frontend/src/domain/pendingProceeding.ts

```diff
@@ -26,10 +38,10 @@ export default class PendingProceeding {
   public caselawReferences?: Reference[]
   public literatureReferences?: Reference[]
   public resolutionNote?: string = ""
-  public isResolved: boolean = false
   public legalIssue?: string = ""
   public admissionOfAppeal?: string = ""
   public appellant?: string = ""
+  public isEditable?: boolean
```

Comment:
das haben wir in der documentUnit.ts auch schon. Kann das hier weg? Das isEditbale ist ja ein attribut, was alle haben